app/ui/navigationSvc.js

```diff
@@ -33,16 +42,24 @@ function navigationSvc($location, $log, stateSvc, appConfig) {
     if (previousChapter > 0) {
       // Go to previous
       $log.info("Going to the Chapter ", previousChapter);
-      PubSub.publish("changingChapter", thisChapter - 1, previousChapter - 1); // (-1 because indeces start at 1)
+      // (-1 because indexes start at 1)
```

Comment:
1 or 0?

Reply:
Could we fix the fact that chapters start at index 1 and properly start them at index 0?